app/controllers/greetings_controller.rb

```diff
@@ -1,6 +1,6 @@
-# class GreetingsController < ApplicationController
-#   def index
-#     @greeting = Message.order('RANDOM()').first.greeting
-#     render json: { greeting: @greeting }
-#   end
-# end
+class GreetingsController < ApplicationController
```

Comment:
- [ ] Kindly fix this error when I run both projects I get this error
![image](https://user-images.githubusercontent.com/24703171/236629297-1e5aa4b6-5f3a-4641-8e8d-4d662d690385.png)

# Hint:
- You can fix it by enabling cors in the backend repo.

Reply:
- [ ] Well resolved 👍🏽 
<img width="389" alt="Screenshot 2023-05-06 at 18 14 01" src="https://user-images.githubusercontent.com/58692590/236638185-3ad4e513-33cd-455b-ab91-4e36450a2bab.png">